### PR TITLE
config export command

### PR DESCRIPTION
**Pull request by laverya**
#### What type of PR is this?

kind/enhancement

#### What this PR does / why we need it:
This PR adds a new command `kots get config` that exports the config values of an application.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
add `kubectl kots get config` command to export config values
```

#### Does this PR require documentation?

https://github.com/replicatedhq/kots.io/pull/625

### CODE REVIEW

**Comment by laverya** [comment]
I'm still working to add a config decryption flag, coming soon ™️

**Comment by laverya** [comment]
also this is not the right package for https://github.com/replicatedhq/kots/blob/c1644141324a74ebecbe0d403d678104d7e43bdd/pkg/handlers/config.go#L998

File: cmd/kots/cli/get-config.go [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+		TypeMeta: v1.TypeMeta{
+			APIVersion: "kots.io/v1beta1",
+			Kind:       "ConfigValues",
+		},
+		Spec: v1beta1.ConfigValuesSpec{Values: map[string]v1beta1.ConfigValue{}},
+	}
+
+	for _, group := range groups {
+		for _, item := range group.Items {
+			extractedValues.Spec.Values[item.Name] = v1beta1.ConfigValue{
```
Comment (jeffreygolden) [inline cmd/kots/cli/get-config.go:158]: This doesn't appear to handle Variadic Config.  I'm not sure if we supported it with this command previously, but we should enter a new ticket to enhance this if it's not resolved in this PR

Comment (laverya) [reply cmd/kots/cli/get-config.go:158]: this does not support variadic config at present, you're right

**Review by morningvera — APPROVED** [review]

**Pull request merged**